### ISSUE TITLE
finish_install.sh in cap-rachel-configure.sh

### ISSUE THREAD

**Issue by needlestack** [rachelproject/rachelplus#77]
Right now `cap-rachel-configure.sh` greps a module's `finish_install.sh` for the line `# Execute this script` to determine if it should execute the script. Instead it should always execute it if it is there.

**Comment by spatiald** [comment]
Yup, good catch - that and several other updates are incorporated but not pushed up to GitHub.  Traveling but will push them once I test for QA.

________________________________
